site/reference/fn/eval/README.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: "Run"
-linkTitle: "run"
+title: "Eval"
```

Comment:
Doing mechanical changes here is not sufficient. Need to rewrite part of this given fundamental change to have this be scoped to imperative use case. Can you file an issue for making this change across all command reference documentation? You don't need to worry about guides since they'll be removed.

@droot

Reply:
Actually, instead of doing piecemeal changes, have a separate PR that updates all reference docs.

Reply:
Agree. We will do one-shot update of the reference docs for each of the subcommand part of `kpt fn`. Keep the docs changes to minimum needed (or keep them as they are in this PR for now) and lets focus on getting the code changes.

Reply:
Added an issue to track this https://github.com/GoogleContainerTools/kpt/issues/1565